### PR TITLE
refactored segments to links + fixed ambiguity of gap closing 

### PR DESCRIPTION
**Pull request by pcmasuzzo**
Renamed segments to **links** and segments table to **links table**
Fixed ambiguity of gap-closing event with the addition of three possible **links_table** representations and the repetition of the **Object_ID** reference, as in #6 
 Also in this PR   - *A split event with a gap closing*

### CODE REVIEW

File: v0.1/index.md [hunk 1]
```diff
@@ -95,23 +95,52 @@ Object_ID  |  Frame  |  X |  Y
...
  1         |    2
  2         |    3
  2         |    4
+ 3         |    2
+ 3         |    3
+
+In these three representations, the gap-closing event is explicitly represented in the repetition of the **Object_ID** reference:
+
+- in **A** and **B**, the **Object_ID** repeated is the one after and before the event, respectively
+- in **D**, both the **Object_ID** of the objects right before and after the event are repeated
```
Comment (sbesson) [inline v0.1/index.md]: C ?

**Review by sbesson — COMMENTED** [review]
One minor comment inline. Overall the tables structural changes with the three possible layouts for the gap closing scenario are in agreement with the output of the public discussion in https://github.com/CellMigStandOrg/Tracks/issues/6. 

One semantics question: following the suggestion of https://github.com/CellMigStandOrg/Tracks/issues/6#issuecomment-283517597, the term `segment` have been replaced to use `links`. My main concern with this change is that the notion of `link` is central and well-defined in graph theory and refers to a non-loop edge connecting two vertices only. In other terms, a CMSO link as proposed in this PR corresponds to a collection of connected graph links.

My personal preference would be to avoid potential mismatch with existing terminologies especially since graphs might be a valid representation of tracks that we could consider at some point.

File: v0.1/index.md (hunk 1 above)
Comment (pcmasuzzo) [reply v0.1/index.md]: C indeed. Mistake!

**Comment by pcmasuzzo** [comment]
As for nomenclature issue raised by @sbesson, I see the problem, but I still suggest we rename `segments` into `links`. The term will have a CMSO annotation anyway (`CMSO:link`), and a dedicated wiki/dictionary will explain the term.

In a perfectly possible (future) representation of tracks through a `graph`, the term `link` could then indeed represent a non-loop edge (as [here](https://proofwiki.org/wiki/Definition:Link_(Graph_Theory))).

I am of course open to other suggestions, but right now I cannot come up with anything better and/or more explicative than this nomenclature.

**Comment by sbesson** [comment]
NB: having briefly looked for alternatives, similar concepts in graph theory would either be `(simple) path` or `chain`.

**Comment by pcmasuzzo** [comment]
So you suggest we use one of those for this (tabular) specification?

**Comment by sbesson** [comment]
Discussed the current proposal with @pcmasuzzo. On the naming topic raised in https://github.com/CellMigStandOrg/Tracks/pull/8#issuecomment-284733825, I have a slight personal preference for `path` but effectively we need additional community input to make these decisions. The current proposal is fine in the context of `v0.1`.

The rest of the changes here make sense and enumerate the various solutions supported in `v0.1` to address the ambiguous case raised in https://github.com/CellMigStandOrg/Tracks/issues/6.

https://github.com/CellMigStandOrg/Tracks/issues/9 has also been opened to explicitly state the optional nature of the tracks table. Merging this so that we can freeze the first version of the specification and focus our efforts towards the data package format and library.

**Pull request merged**